FundacionParaguaya/MentorApp
# 1099 conditional checkboxes

**Problem** (SocioEconomic screen)
1. In survey Paraguay - Testing Conditionals , currently the options for the House situations were not being loaded conditionally. 

2. Also when you try to navigate from the 'Family Data' screen to the next one, the app was crashing due to not properly handling how the screens are loaded/skipped based on conditional answers from the survey
(the same survey Paraguay, Testing Conditionals)

**Describe the changes**
Changed the props which are passed to the Checkboxes component, so they can be loaded conditionally. Removed updateChckbox method inside SocioEconomic question and made the Checkboxes component update using the existing updateEconomicAnswer method. 

Added additional check in the method responsible for navigating to the next screen number.

**To Test**
1. Start 'Paraguay - Testing conditionals'
2. When you reach the Family data screen try changing answers for the question "Area where the family lives"
3. You should see options being loaded in the next question "Housing situation", based on the answer of the previous question
4. Press "continue"  and try navigating to the next screen
5. It should not crash anymore

This will also fix #1098
